BusinessLayer/InventoryCsvManager.cs

```diff
@@ -25,7 +25,7 @@ public Contact AddSerialNumber(Contact contact)
             contact.SerialNumber = GetLastSerialNumber() + 1;
             using (var streamWriter = new StreamWriter(_config["INVENTORY_PATH"], true))
             {
-                var line = "\n" + contact.SerialNumber + "," + contact.Name + "," + contact.Mail + "," +
+                var line = "\n" + contact.SerialNumber + "," + contact.Name + "," + contact.Mail + "," + _config["CONFERENCE_NAME"] +
```

Comment:
the date of the conference is still missing

Reply:
The changes request are here, there is just the date which is on the following line

![image](https://user-images.githubusercontent.com/60068695/96556111-df9fb400-12b8-11eb-8768-6682a68dbfae.png)

Reply:
no, this is the date of the emitssion of the certificate, not the date of the conference. you need two different dates